powerbi-docs/transform-model/datamarts/datamarts-understand.md

```diff
@@ -12,34 +12,44 @@ LocalizationGroup: Data from files
 ---
 # Understand datamarts
 
-This article describes and explains important concepts about datamarts.
+## Understand Dataset (default)
```

Comment:
Blocking issue: An intro paragraph must appear between the H1 and the first H2.

Reply:
we can add this sentence back in.